src/System.Memory/tests/SequenceReader/Rewind.cs

```diff
@@ -72,13 +72,32 @@ public void Rewind_ByOne()
         public void Rewind_Exception()
         {
             ReadOnlySequence<byte> bytes = SequenceFactory.Create(new byte[][] {
-                new byte[] { 0          },
```

Comment:
Are we losing coverage when we change this input? We should add back a test with this input.

Reply:
We aren't. Having more than 2 segments for this test distracts from what we're testing (which never crosses segments). I'll add a test that crosses and rewinds should anyone tinker with the basic logic here.